src/clientiface.cpp

```diff
@@ -90,13 +90,30 @@ void RemoteClient::GetNextBlocks (
 		return;
 	}
 
+	// get view range and camera fov from the client
+	s16 wanted_range = sao->getWantedRange();
+	float camera_fov = sao->getFov();
+
+	// if FOV, wanted_range are not available (old client), fall back to old default
```

Comment:
We're breaking compat with old clients now, so old clent support can be dropped.